dspace/config/modules/curate.cfg

```diff
@@ -29,3 +29,61 @@ curate.taskqueue.dir = ${dspace.dir}/ctqueues
 
 # Maximum amount of redirects set to 0 for none and -1 for unlimited
 curate.checklinks.max-redirect = 0
+
+# The CopyFieldsCurationTask copies one or more metadata fields into one target field.
+# It is highly configurable. As you might want to run multiple instances of it, it uses
+# the curation task name as defined in the named plugin configuration to lookup its configuration
+# (see the examples below).
+#
+# The CopyFieldsCurationTask can work on Communities, Collections, Items and Bitstreams.
+# Configure on which of those objects metadata should actually be changed or which objects
+# should only be traversed when looking for other objects, by setting
+# curate.<ctask-name>.types to a comma separated list of dspace object types, for example:
+# curate.<ctask-name>.types = Item, Bitstream
+# if you want it to change metadata of items and of bitstreams, but not on communities
+# and collections. You can still change all items and bitstreams included in a collections
+# by starting the curation task on that collection
+# ('/dspace/bin/dspace curate -t <ctask-name> -e <eperson-email-address> -i <collection-handle>').
+#
+# You must configure one target field:
+# curate.<ctask-name>.sourceFields = dc.contributor.other
+# Please make sure, that the target field exist in your DSpace installation.
+#
+# You must configure a list of one or more source fields. The order of that list will be respected,
+#  when copying their values into the target field:
+# curate.<ctask-name>.sourceFields = dc.contributor.author, dc.contributor.editor, dc.contributor.referee
+#
+# You can configure a concatenator that will be used to concatenate the values of different source fields.
+# Please put this into single quotes, as we try to catch any space you include into the concatenator.
+# If you use commas, you must escape those:
+# curate.<ctask-name>.concatenator = '\, '
+# By default no concatenator is used, and fields will be merge together without any separation between them.
+# You can configure if the concatenator shall be added after the last value (by default it is not). This
+# might be useful, if you use a dot as concatenator:
+# curate.<ctask-name>.cocatenatorAtTheEnd = true
```

Comment:
I think we're missing an automated test for the `concatenatorAtTheEnd` configuration.

Also, you've accidentally misspelled it in this example.  You're missing the "n" in "con".  It should be.
```
curate.<ctask-name>.concatenatorAtTheEnd = true
```